api-extractor/src/Extractor.ts

```diff
@@ -95,7 +95,8 @@ export default class Extractor {
       throw new Error('Unable to load file: ' + options.entryPointFile);
     }
 
-    this.package = new ApiPackage(this, rootFile);
+    this.package = new ApiPackage(this, rootFile); // construct members
+    this.package.resolveReferences(); // creates ApiDocumentation
```

Comment:
We don't this change is contained within the Extractor.analyze() function so it is hidden to the Gulp task.